packages/spec/src/tools/html.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright IBM Corp. 2019, 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import htm from 'htm';
+
+const nsTags = new Set([
+  'circle',
+  'defs',
+  'desc',
+  'ellipse',
+  'g',
+  'line',
+  'path',
+  'pattern',
+  'polygon',
+  'rect',
+  'style',
+  'svg',
+  'text',
+  'textPath',
+  'title',
+  'use',
+]);
+
+export function createElement(tagName, attributes = {}, ...children) {
```

Comment:
Am I correct that this function is only used by the tests of the `diff` function?

Reply:
Yup!